tsconfig.json

```diff
@@ -2,16 +2,22 @@
   "compilerOptions": {
     "allowSyntheticDefaultImports": true,
     "allowUnreachableCode": false,
+    "baseUrl": "./src",
     "declaration": false,
     "experimentalDecorators": true,
+    "jsx": "react",
+    "jsxFactory": "h",
     "lib": ["dom", "es2017"],
-    "moduleResolution": "node",
     "module": "esnext",
-    "target": "es2017",
+    "moduleResolution": "node",
+    // "noImplicitAny": true,
```

Comment:
I would like to enable this, but it seems if I do, manifold-init has type errors 😕